code/modules/admin/verbs/randomverbs.dm

```diff
@@ -828,6 +828,23 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	message_admins("[key_name(src)] cleaned air within [size] tiles.")
 	log_game("[key_name(src)] cleaned air within [size] tiles.")
 
+/client/proc/fix_gravity()
+	set name = "Turn on gravity"
+	set category = "Special Verbs"
+	set desc = "Use this when griefons turn off the gravity gen "
+	for(var/obj/machinery/gravity_generator/main/grav in world)
+		if(grav.z == 1) // make sure the grav gen is in z 1
+			grav.breaker = 1
+			grav.charge_count = 100
+			grav.set_state(1) // Turn it on!!

```

Comment:
I haven't looked in the gravity generator code but I'd be willing to bet that there's atleast an update_icon function which would do the job of updating the overlays for you.